features/help.feature

```diff
@@ -62,15 +70,186 @@ Feature: Get help about WP-CLI commands
     Then the return code should be 1
     And STDERR should be:
       """
-      Error: 'non-existent-command' is not a registered wp command.
+      Error: 'non-existent-command' is not a registered wp command. See 'wp help'.
```

Comment:
What do you think about changing:

> See 'wp help'.

To:

> See 'wp help' for available commands.

I think it adds more clarity to the message.